go.mod

```diff
@@ -35,6 +34,9 @@ require (
 	github.com/swaggo/gin-swagger v1.2.0
 	github.com/zsais/go-gin-prometheus v0.1.0
 	golang.org/x/net v0.0.0-20201029221708-28c70e62bb1d
+	gorm.io/datatypes v1.0.0
```

Comment:
Is it better to use gorm.io then github?

Reply:
gorm.io is v2, so I think it's better to use newest package, but it's not user anywhere, so I will delete it.